queries/highlights.scm

```diff
@@ -1,16 +1,13 @@
-[
-    (comment)
-    (note)
-] @comment
+((comment) @comment)
+((note) @comment.note)
 
-[
-    (date)
-    (interval)
-    (quantity)
-] @number
+((date) @number.date)
+((interval) @number.interval)
+((quantity) @number.quantity)
+((quantity_negative) @number.negative.quantity)
```

Comment:
Apart from this line the changes in this file are proposed in #21

Reply:
I think that `@number.quantity.negative` makes more semantic sense